src/components/App.css

```diff
@@ -175,6 +175,16 @@ input[type="number"] {
 }
 
 .battle-toolbar--button {
+  padding: 4px;
+  height: 40px;
+  margin-top: 25px;
+}
+
+.battle-toolbar--button__top, .battle-toolbar--button__progress {
+  margin-top: 5px !important;
```

Comment:
Avoid using `!important` if at all possible. I removed it in the browser and it doesn't seem to be necessary.